kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UIServicesClientImpl.java

```diff
@@ -210,6 +212,21 @@ public String getProcessInstanceImage(String containerId, Long processInstanceId
         }
     }
 
+    protected String createColorURLParams(String completeNodeColor, String completeNodeBorderColor, String activeNodeBorderColor){
```

Comment:
please resolve white-space issues in the newly added code (missing space between ')' and '{', missing space around '=', no spacing around '!=', ...), reformat according to code style.

Reply:
@jstastny-cz updated